guides/v2.3/extension-dev-guide/message-queues/config-mq.md

```diff
@@ -27,6 +28,48 @@ Depending on your needs, you may only need to create and configure `communicatio
 * In cases where you want to configure the local queue and publish to it for 3rd party systems to consume, you will need the `queue_publisher.xml` and `queue_topology.xml` files.
 * When you want to configure the local queue and consume messages published by 3rd party system, you will need the `queue_topology.xml` and `queue_consumer.xml` files.
 
+### `queue.xml` {#queuexml}
+
+The `queue.xml` file defines the broker that processes topics. It also specifies the queue each topic will be sent to.
+
+### Sample `queue.xml` file
+{:.no_toc}
+
+```xml
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Communication/etc/communication.xsd">
+    <broker topic="product_action_attribute.update" exchange="magento-db" type="db">
+        <queue name="product_action_attribute.update" 
+               consumer="product_action_attribute.update" 
+               consumerInstance="Magento\Framework\MessageQueue\Consumer" 
+               handler="Magento\Catalog\Model\Attribute\Backend\Consumer::process"/>
+    </broker>
+</config>
+```
+
+### broker element
+{:.no_toc}
+
+The `broker` element also contains `queue` elements.
+
+Parameter | Description
+--- | ---
+topic | A topic defined in the `communication.xml` file.
+type | The type of message broker. For this release, the value must be `amqp` or `db`.
```

Comment:
Should the value be `db`, since RabbitMQ connections don't use this file?

Delete "For this release, "

Reply:
**fixed**

Reply:
Thank you, @keharper!